locator/topology.hh

```diff
@@ -139,6 +141,16 @@ public:
         _excluded = excluded;
     }
 
+    // Drained nodes are about to be decommissioned and tablet scheduler
+    // moves tablet replicas away from them.
```

Comment:
Not sure if I understand this correctly at this point of my review, but it looks like only a normal node that is about to be decommissioned can be drained. Shouldn't drain be a value of `node::state` then?